frontend/integration-tests/tests/limitrange.scenario.ts

```diff
@@ -0,0 +1,47 @@
+import { browser, $$, ExpectedConditions as until } from 'protractor';
+import { safeLoad } from 'js-yaml';
+import { appHost, testName } from '../protractor.conf';
+import * as crudView from '../views/crud.view';
+import * as yamlView from '../views/yaml.view';
+
+
+describe('Limitrange test', () => {
+  let defaultLimit = '';
+  let defaultRequest = '';
+  let type = '';
+  it(`create limitrange resource for ${testName}`, async() => {
+    await browser.get(`${appHost}/k8s/ns/${testName}/limitranges`);
+    await crudView.createYAMLButton.click();
+    await yamlView.isLoaded();
+    const yamlContent = await yamlView.editorContent.getText();
+    const content = safeLoad(yamlContent);
+    for (const limitType of content.spec.limits){
+      defaultLimit = limitType.default.memory;
+      defaultRequest = limitType.defaultRequest.memory;
+      type = limitType.type;
+    }
+    await crudView.saveChangesBtn.click();
+    await browser.wait(until.presenceOf($$('.co-m-table-grid__head tr td').get(0)));
+    const headers = $$('.co-m-table-grid__head td');
+    const values = $$('.co-resource-list__item td');
+    headers.then( items => {
+      expect(items[0].getText()).toBe('TYPE');
```

Comment:
This would be better as an enzyme test like

https://github.com/openshift/console/blob/master/frontend/__tests__/components/route-pages.spec.tsx

Reply:
@spadgett Thanks for suggest. But do you mean to move the columns checking to Unit test? Then this PR could be closed because the value is in same column with header.

Reply:
Yes, I'm saying this is better as a unit test than an integration test. Either way, this will need to be reworked for #1465 since all tables are about to change.

The simple way to get basic support for LimitRange in our integration tests is to add it to the existing crud test resources: https://github.com/openshift/console/blob/master/frontend/integration-tests/tests/crud.scenario.ts#L20-L40